package/classes/EventBus.cls

```diff
@@ -10,10 +69,10 @@ public class EventBus
     /**
      * @class Event
      * Represents a fired event.
-	 * @property {String} Name The name of the event invoked
-	 * @property {Map<String, Object>} Arguments The arguments passed with this invocation
-	 * @property {List<String>} Errors A list of error messages caused by the listeners of this event
-	 * @property {Boolean} Veto True if this event has been terminated
+     * @property {String} Name The name of the event invoked
```

Comment:
There are a lot of whitespace changes, make sure all files are using tabs for indentation, not spaces.

- [ ] Ensure tabs are used for indentation